mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -458,6 +458,7 @@ void eventBusShouldNotThrowWhenContinuouslyStartAndStop() {
                 }).doesNotThrowAnyException();
             }
 
+            @Disabled("JAMES-2569 unstable test")
```

Comment:
this is 2569: https://issues.apache.org/jira/browse/JAMES-2569 . I don't see how it is related?

Reply:
I missed read it, it should be https://issues.apache.org/jira/browse/JAMES-2659